tf/env/staging/namespaces.tf

```diff
@@ -66,3 +66,9 @@ resource "kubernetes_resource_quota" "qs-jobs-podquota" {
     scopes = ["BestEffort"]
   }
 }
+
+resource "kubernetes_namespace" "tailscale-namespace" {
```

Comment:
IIRC helmfile will do this for us when deployng the tailscale operator the tailscale namespace. The namespaces we create in terraform are currently for use cases where there's no release tied to the namespace, e.g. for running jobs.

Reply:
Ah, got it.